src/request/TopLevelApi.js

```diff
@@ -118,9 +118,18 @@ class TopLevelApi { // eslint-disable-line no-unused-vars
         this._reject(error);
     }
 
+    static setLoading() {
+        document.body.classList.add('loading');
+    }
+
+    static removeLoading() {
+        document.body.classList.remove('loading');
+    }
+
```

Comment:
As a suggestion for both in one function.
```javascript
static loading(loading=true) {
  document.body.classList.toggle('loading',loading);
}
```

Reply:
+1

Reply:
I would keep setLoading as name, though.

Reply:
I disagree and actually made it two functions on purpose, to make it more explicit and not have to pass a boolean for removing it.

Reply:
I personally am in favour of one function for that kind of functionality. More so as `classList.toggle` exists.